pennylane/qchem/structure.py

```diff
@@ -412,3 +413,130 @@ def excitations_to_wires(singles, doubles, wires=None):
         doubles_wires.append([d1_wires, d2_wires])
 
     return singles_wires, doubles_wires
+
+
+def mol_data_from_pubchem(identifier, identifier_type="name", cid=None):
+    r"""Obtain molecular data of the compound from the PubChem Database
```

Comment:
Better to be specific about data, it means name, geometry and charge, right?

Also important to mention that the geometry is in Bohr, with a link.